ui/src/components/Inventory/InventoryFormModal.js

```diff
@@ -0,0 +1,158 @@
+                      name='productType'
+                      label='Product Type'
+                      component={TextField} select
+                      required
+                      validate={validateRequired}
+                    >
+                      {products.map(product => (
+                        <MenuItem key={product.id} value={product.name}>
+                          {product.name}
+                        </MenuItem>
+                      ))}
+                    </Field>
+                  </Grid>
+                  <Grid item xs={12} sm={12}>
+                    <Field
+                      custom={{ variant: 'outlined', fullWidth: true, }}
+                      name='description'
+                      label='Description'
+                      component={TextField}
+                    />
+                  </Grid>
+                  <Grid item xs={12} sm={12}>
+                    <Field
+                      custom={{ variant: 'outlined', fullWidth: true, }}
+                      name='averagePrice'
+                      label='Average Price'
+                      component={TextField}
+                      type='number'
+                      validate={validatePositiveNumber}
+                    />
+                  </Grid>
+                  <Grid item xs={12} sm={12}>
+                    <Field
+                      custom={{ variant: 'outlined', fullWidth: true, }}
+                      name='amount'
+                      label='Amount'
+                      component={TextField}
+                      type='number'
+                      validate={validatePositiveNumber}
+                    />
+                  </Grid>
+                  <Grid item xs={12} sm={12}>
+                    <Field
+                      custom={{ variant: 'outlined', fullWidth: true, }}
+                      name='unitOfMeasurement'
+                      label='Unit of Measurement'
+                      component={TextField} select
+                      required
+                      validate={validateRequired}
+                    >
+                      {Object.keys(measurementUnits).map(measurementUnit => (
+                        <MenuItem key={measurementUnit} value={measurementUnit}>
+                          {measurementUnits[measurementUnit].name}
+                        </MenuItem>
+                      ))}
+                    </Field>
+                  </Grid>
+                  <Grid item xs={12} sm={12}>
+                    <Field
+                      custom={{ variant: 'outlined', fullWidth: true, }}
+                      name='bestBeforeDate'
+                      label='Best Before Date'
+                      component={TextField}
+                      type='date'
+                    />
+                  </Grid>
+                  <Grid item xs={12} sm={12}>
+                    <FormControlLabel control={<Checkbox/>} label="Never Expires" name = 'neverExpires'/>
```

Comment:
Change this into a formik  `<Field />` using with the following properties:
```
custom={{ variant: 'outlined', fullWidth: true, }}
as={FormControlLabel}
control={<Checkbox />}
name='neverExpires'
label='Never Expires'
type="checkbox"
```

Reply:
Change made in commit 34e7176e11f9d1cfb470e582f113750362cb1e02